README.md

```diff
@@ -256,3 +257,27 @@ make load-fixtures
 
 Follow installing go + vscode [setup guide](https://marketplace.visualstudio.com/items?itemName=golang.go#getting-started).
 Additional settings found under `.vscode/settings.json` allow tests to be run within vscode.
+  
+# Installing and Using Pre-commit
```

Comment:
can we move this to the top (under the "Dependencies" section) and/or link to the section from the Dependencies list?

Reply:
same comment for the other repos — if we could keep this near the top so people don't miss it and forget to use pre-commit, that'll be helpful.